GOFevaluation/utils.py

```diff
@@ -224,12 +331,19 @@ def plot_equiprobable_histogram(data_sample, bin_edges, order=None,
         z-axis in units of sigma-deviation from expectation. If None is passed,
         cmap scale ranges from min to max. Defaults to None.
     :type nevents_expected: float, optional
+    :param plot_mode: sets the plotting schedule. defaults to sigma_deviation.
+        can be set to 'num_counts' or 'count_density' (if not one of these,
+        sigma_deviation method ) will be used.
+    :type: string 'num_counts' or string 'count_density'
```

Comment:
the sentence within the parentheses is already implied above.
:type: is just string.
Maybe it would make sense to instead put a one-sentence-description of each of the three modes (formatted like the order kwarg)